packages/hub/src/components/SquiggleSnippetContent/ViewSquiggleSnippetContent.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from "react";
+import ReactMarkdown from "react-markdown";
+import { useFragment } from "react-relay";
+import remarkBreaks from "remark-breaks";
+
+import { SquiggleChart } from "@quri/squiggle-components";
+
+import { SquiggleSnippetContentFragment$key } from "@/__generated__/SquiggleSnippetContentFragment.graphql";
+import { Fragment } from "./SquiggleSnippetContent";
+
+export const ViewSquiggleSnippetContent: FC<{
+  model: SquiggleSnippetContentFragment$key;
+}> = ({ model }) => {
+  const data = useFragment(Fragment, model);
+
+  if (data.currentRevision.content.__typename !== "SquiggleSnippet") {
+    // shouldn't happen, typename is validated by ModelView
+    throw new Error("Internal error");
+  }
+
+  return (
+    <div>
+      {data.currentRevision.description === "" ? null : (
+        <div className="mb-4">
+          <ReactMarkdown
+            remarkPlugins={[remarkBreaks]}
+            components={{
+              h1: ({ node, ...props }) => (
+                <h1 className="text-2xl font-medium" {...props} />
+              ),
+              h2: ({ node, ...props }) => (
+                <h1 className="text-xl font-bold" {...props} />
+              ),
+              h3: ({ node, ...props }) => (
+                <h1 className="font-bold" {...props} />
+              ),
```

Comment:
minor note, but I think there might be some Tailwind libs or something to help deal with markdown.

Reply:
There is https://tailwindcss.com/docs/typography-plugin, but I wanted to have more fine-grained control over styles here (e.g. for links it would be necessary anyway). And that plugin is optimized for longer blog posts, we probably want a more dense layout for descriptions.

Reply:
Yea, makes sense, at least for now. Can always add more later.